src/input/user_input.rs

```diff
@@ -830,8 +832,7 @@ where
                     <label for="input-file" class="input-file-choose-file">
                         { text!(txt, ctx.props().language, "Choose a file") }
                     </label>
-
-                    <input type="file" id="input-file" accept=".aice" name="upload-file" style="display: none;"
+                    <input type="file" id="input-file" accept={extensions.clone()} name="upload-file" style="display: none;"
```

Comment:
Is clone necessary here?

Reply:
Thank you for catching that! I removed the .clone() as it seems unnecessary. Let me know if you'd like me to make any further adjustments!